cset160-2-bare-bones-ruby-master/db/migrations/1_create_students_table.rb

```diff
@@ -6,17 +6,17 @@
   create table students (
```

Comment:
_table name tips:_ This is the **first** spot you would want to change. Once you change here, you would want to re-run `ruby db/migrate.rb`